test/unstable/const-intent/flags/checkFlags.skipif

```diff
@@ -0,0 +1,2 @@
+# this test checks whether --fast impacts it, so don't run when it is set
+COMPOPTS <= --fast
```

Comment:
Does this also apply to `--warn-unstable` ?

Reply:
We don't have a job for `--warn-unstable`, tests of it only live in the `test/unstable` directory, so we have complete control over whether it will be used when this test is run.  If we move to having a `--warn-unstable` job in the future, I think it would be reasonable to add then.